src/main/public/js/analytics-custom-event-sender.js

```diff
@@ -67,7 +67,10 @@ $(function () {
   // Send a google analytics event when an element that has the 'analytics-click-event-trigger' class is clicked.
   $('.analytics-click-event-trigger').on('click', function () {
     var cookies_policy = getCookie("cookies_policy");
-    var json = JSON.parse(this.response);
+    var json = null;
+    if (typeof this.reponse !== 'undefined' || this.response !== undefined){
```

Comment:
this.reponse instead of this.response
Also, JSON.parse(null) does not fail? why not use if (this.response) json = JSON.parse(this.response)

Reply:
And also, uses of json from this point on need null check, like lines 78, 80 when json.gaTrackingId is called